README.md

```diff
@@ -68,3 +68,6 @@ Instructions to enable file logging:
 
 - Code and scripts inside the folder `src\vpk_install` are licensed under the terms of GNU GPLv3. Check the file `src/vpk_install/license.txt` for the complete conditions.
 - The rest of the files (unless specified otherwise) are licensed under the terms of [CC by-nc-sa 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/). Check the file `license.txt` for the complete conditions.
+
+## Thanks to
+- devingDev for some search [dialog code](https://github.com/devingDev/VitaCord/blob/master/src/VitaIME.cpp).
```

Comment:
I suggest removing the credit as they are not the author of this.

Reply:
Who is the author then? I'm mean I literally copied his code and customized it.

Reply:
I'm not exactly sure but this might be it: https://github.com/TheOfficialFloW/VitaShell/blob/master/ime_dialog.c

The utf conversion part seems to be heavily inspired from Stackoverflow.